app/editor/src/store/slices/app/appSlice.ts

```diff
@@ -0,0 +1,29 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+import { IAppState } from '.';
+
+/**
+ * The following is a shorthand method for creating a reducer with paired actions and action creators.
+ * All functionality related to this concept is contained within this file.
+ * See https://redux-toolkit.js.org/api/createslice for more details.
+ */
+export const appSlice = createSlice({
```

Comment:
Used for generic app level redux state.  Currently to track if we're loading an AJAX request.